examples/amp-web-push.amp.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
```

Comment:
For this commit, I changed the committer email to match that listed on the Google CLA.